### ISSUE TITLE
Milestone - Integrate the NMDC Submission Portal into User Facility processes (1.1)

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#444]
Automation and coordination with JGI & EMSL User Facilities
The JGI and EMSL are key partners for the NMDC because they support the environmental research
community, have established processes for collecting metadata from researchers, and execute nearly all
of the NMDC workflows as part of their data generation process. The NMDC collaborated closely with the
User Facility staff during the Pilot to incorporate the multi-omics data generated from proposals
supported in the Facilities Integrating Collaborations for User Science (FICUS) Program57. These studies
were a natural target because the FICUS projects include a number of exciting investigations where multi-
omics data was generated from the same sample, making them a natural fit for NMDC. Building on this
successful engagement will allow us to reach our Renewal aims. We will leverage the User Facility
partnerships to enhance and integrate the Data Submission Portal in the User Facility processes
(Milestone 1.1), with co-development of APIs to access the appropriate metadata from the respective JGI
and EMSL data management systems (Figure 4, D and E; Milestone 1.2), and automate processed multi-
omics data sharing from the User Facilities (Figure 4, B and D; Milestone 1.3).

Pages 22-23

**Comment by aclum** [comment]
On the JGI side MOnet, NEON, and Island Microbiome will use NMDC submission portal. 
@mslarae13  will any EMSL projects be ready to submit via the nmdc submission portal this quarter?

**Comment by mslarae13** [comment]
https://github.com/microbiomedata/metadata-template-transformation/issues/16

**Comment by mslarae13** [comment]
> will any EMSL projects be ready to submit via the nmdc submission portal this quarter?

@aclum  I've encouraged a couple projects to. But no guarantee yet. One of them Amy Zimmerman is the PI & it's a project related to the EMSL Soil SFA.

**Comment by ssarrafan** [comment]
Update from quarterly report:  As part of the User Facilities Optimization (UFO) squad, we have collaborated with JGI Project Managers to design a workflow for users to submit metadata via the NMDC Submission Portal with less manual intervention. After several iterations, a workflow has been agreed upon and will be finalized FY24 Q2. This work includes a ‘reviewer’ view for JGI Project Managers and a GOLD curator; an API into JGI’s sample submission system to access JGI-provided metadata to the user in the NMDC Submission Portal; and interface development to collect required JGI and API access information with correct sample and metadata association.

For EMSL users able to submit metadata via the NMDC Submission Portal, we have expanded the NMDC schema to include additional EMSL requirements for users with soil, sediment, plant, and water samples.

**Comment by ssarrafan** [comment]
From DOE Quarterly report:
As part of the User Facilities Optimization (UFO) squad, we have collaborated with JGI Project Managers to design a workflow for users to submit metadata via the NMDC Submission Portal with less manual intervention. After several iterations, a workflow has been agreed upon and will be finalized FY24 Q2. This work includes a ‘reviewer’ view for JGI Project Managers and a GOLD curator; an API into JGI’s sample submission system to access JGI-provided metadata to the user in the NMDC Submission Portal; and interface development to collect required JGI and API access information with correct sample and metadata association.

For EMSL users able to submit metadata via the NMDC Submission Portal, we have expanded the NMDC schema to include additional EMSL requirements for users with soil, sediment, plant, and water samples.
